projects/fsastorefrontstyles/scss/components/checkout/_mini-cart.scss

```diff
@@ -31,3 +31,37 @@
     }
   }
 }
+
+.action-wrapper {
+  display: flex;
+  justify-content: space-between;
+  cx-fs-cart-coupon {
+    width: 60%;
+  }
+  cx-fs-cms-sync-pilot {
+    width: max-content;
```

Comment:
If possible, we should use flex values here as well.

Reply:
Same here: It seems there is no need for the width of flex.